web/packages/teleport/src/services/api/api.ts

```diff
@@ -18,6 +18,7 @@
 
 import 'whatwg-fetch';
 import auth, { MfaChallengeScope } from 'teleport/services/auth/auth';
+import websession from 'teleport/services/websession';
```

Comment:
This is another cyclic dependency in this file which might cause problems in the future.

`teleport/services/websession` depends on `teleport/services/api` and `teleport/services/api` now depends on `teleport/services/websession`.